hidapi/hidapi.h

```diff
@@ -132,6 +163,8 @@ extern "C" {
 				* Valid on the Mac implementation if and only if the device
 				  is a USB HID device. */
 			int interface_number;
+			/** Underlying bus type */
+			hid_bus_type bus_type;
 
 			/** Pointer to the next device */
 			struct hid_device_info *next;
```

Comment:
I'm afraid that having this member here breaks the ABI.
A safer change would be:
```suggestion

			/** Pointer to the next device */
			struct hid_device_info *next;

			/** Underlying bus type
			    Since version 0.13.0, @ref HID_API_VERSION >= HID_API_MAKE_VERSION(0, 13, 0)
			*/
			hid_bus_type bus_type;
```